vrk-kpa/suomidigi
# SUOPA-154 Theme build tools

To test: 
- Test that theme building works by running `make build` or `make fresh`
- Clear the caches and load http://suomidigi.fi.docker.amazee.io/ 
   - You should see partly styled header (Yes, this is a bonus part)
